### ISSUE TITLE
PyPI Deploy failing due to description content type.

### ISSUE THREAD

**Issue by spyoungtech** [spyoungtech/behave-webdriver#77]
This error is strange since the README and setup.py have not changed (AFAIK) since last release. 

Error as seen [on travis](https://travis-ci.org/spyoungtech/behave-webdriver/jobs/534061643)

```
Uploading behave_webdriver-0.3.0-py2.py3-none-any.whl
100%|██████████████████████████████████████| 43.3k/43.3k [00:00<00:00, 70.2kB/s]
NOTE: Try --verbose to see response content.
HTTPError: 400 Client Error: The description failed to render in the default format of reStructuredText. See https://pypi.org/help/#description-content-type for more information. for url: https://upload.pypi.org/legacy/
```

**Comment by spyoungtech** [comment]
twine check reveals an error on line 220 for the `guess` lexer.

Weird that it didn't happen on previous release.... Removing `guess` allows twine check to pass

**Comment by spyoungtech** [comment]
Resolved and 0.3.0 is live on PyPI
